Towards re-enabling unidoc

raboof opened the pull request:
`sbt -Dakka.genjavadoc.enabled=true ++2.12.3 unidoc` seems to produce scaladoc and javadoc now, not sure what to look for beyond that.

(remaining jenkins error is formatting)

ktoso committed:
BUILD CHANGES TO COMPILE ON JDK9 USING SBT 1.0, DO NOT MERGE

raboof committed:
Update whitesource plugin (#23654)

ktoso committed:
rename server

ktoso committed:
perhaps this will work

ktoso committed:
jdk9-build: Update paradox to 0.3.0 (#23665)

raboof committed:
Towards re-enabling unidoc

raboof committed:
Remove fix for #18056 (from akka-http era)

raboof committed:
Remove -P:genjavadoc:fabricateParams=true" setting

As it failed on jenkins, and appears to be the default nowadays anyway

raboof committed:
Enable the 'buildRelease' command again

ktoso commented on project/Release.scala:
```diff
@@ -1,45 +1,44 @@
+    state3
+  }
+
+}
```
exciting :-)

2m commented:
I have tried to build docs with this PR and it seems that javadoc is not generated. Directory `target/javaunidoc` is empty.

raboof committed:
Make javadoc work on jdk9 again

Avoid passing '-target 1.8', which is not supported/allowed
by the javadoc tool.

raboof commented:
Already applied to `jdk9-build`

ktoso commented:
Good:)
